pyproject.toml

```diff
@@ -42,13 +42,13 @@ ignore_missing_imports = true
 
 [tool.poetry]
 name = "spark8t"
-version = "0.0.3"
+version = "0.0.4"
 description = "This project allow you to create a SNAP for interacting with a K8s cluster to submit Spark jobs or run spark shells interactively"
 authors = [
-    "Abhishek Verma <abhishek.verma@canonical.com>",
     "Enrico Deusebio <enrico.deusebio@canonical.com>",
     "Paolo Sottovia <paolo.sottovia@canonical.com>",
-    "Judit Novak <judit.novak@canonical.com>"
+    "Judit Novak <judit.novak@canonical.com>",
+    "Bikalpa Dhakal <bikalpa.dhakal@canonical.com>"
 ]
 license = "MIT"
```

Comment:
Is this the correct license? I thought we generally use ASLv2

Reply:
Yes, you are right! I have placed "Apache-2.0" which is consistent with what we use in operators.